keyboards/leeku/l3_tkl/config.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2021 MajorKoos <https://github.com/majorkoos>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x20A0
```

Comment:
This vendor ID is in use.  Please change it. 

https://www.the-sz.com/products/usbid/index.php?v=0x20A0&p=&n=

Reply:
How about FFFE ?
That seems completely unused.
I wasn't really sure what to do about the vendor ID so I picked something that wasn't taken under clay logic.